qutebrowser/browser/qutescheme.py

```diff
@@ -159,6 +161,35 @@ def qute_bookmarks(_url):
     return 'text/html', html
 
 
+@add_handler('history')
+def qute_history(_url):
+    """Handler for qute:history. Display browsing history."""
+    history = reversed(objreg.get('web-history').history_dict.values())
+
+    fmt = config.get('completion', 'timestamp-format')
+    if fmt is None:
+        def fmt_atime(atime):
+            return ''
+    else:
```

Comment:
if it doesn't make much of a difference, I'd prefer doing the check inside with the `config.get` outside of it. Makes things easier to follow, IMHO.